composer.json

```diff
@@ -32,11 +32,11 @@
         "flow-php/etl-adapter-http": "^0.1 || ^0.2",
         "google/apiclient": "^2.0",
         "phpbench/phpbench": "^1.2.6",
-        "symfony/browser-kit": "~4.4.44||~5.4.21||~6",
-        "symfony/cache": "~4.4.48||~5.1||~6.0",
-        "symfony/dom-crawler": "~4.4.45||~5.4.21||~6.0",
-        "symfony/framework-bundle": "~4.4.49||~5.4.21||~6.0",
-        "symfony/security-bundle": "~4.4.44||~5.4.21||~6.0"
+        "symfony/browser-kit": "^5.4.21||^6.3||^7.0",
+        "symfony/cache": "^5.4||^6.3||^7.0",
+        "symfony/dom-crawler": "^5.4.21||^6.3||^7.0",
+        "symfony/framework-bundle": "^5.4.21||^6.3||^7.0",
+        "symfony/security-bundle": "^5.4.21||^6.3||^7.0"
```

Comment:
this is unecessary, dev dependencies does not need to be tested across all versions as they are not exposed to end users, I think I would use just 6.4 as the latest LTS.